src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search.html

```diff
@@ -1,6 +1,13 @@
 <ng-form class="umb-mini-search" ng-class="{'--has-value': vm.model !== null && vm.model !== ''}" novalidate>
     <i class="icon icon-search" aria-hidden="true"></i>
```

Comment:
Altough not related to this PR we should probably use `<umb-icon>` here instead 😁